docs/comparative-table.md

```diff
@@ -0,0 +1,11 @@
+# Comparative table of similar software
+      
+
+| Software   | Description | Price   |
+| :---        |    :----:   |          ---: |
+| [Anaplan](https://www.anaplan.com)    | "Anaplan PlanIQ™ helps users drive accurate decisions with forecasting that leverages powerful, integrated artificial intelligence (AI) and machine learning (ML). Collaborate across your enterprise, working with a single source of truth, updated and calculated in real-time. Test multiple scenarios and quickly operationalize best options."  | $30,000 per year for small teams |
```

Comment:
TLDR, I'd prefer bullet points with key information rather than a long description

Reply:
And would be way better, instead of bullet points, do columns with key features and maybe mark them with a cross `x` if they have them or not, i.e.:

| Software   | Description | Feature 1   | Feature 2   | ...   | Feature n   | Price   |
| :---        |    :----:   |    :----:   |    :----:   |    :----:   |    :----:   |          ---: |
| Anaplan   | Short desc | x |  | ... | x | $$$ |
| ...   | ... | ... | ... | ... | ... | ... ||
| Software n   | Short desc |  |  | ... | x | $$$ |

Reply:
@rebo-it/reviewers Can you send me the markdown version of this list, thank you

Reply:
Clicking the 3 dots at the right upper corner of a comment let's you see the raw comment
![image](https://github.com/user-attachments/assets/726cf7d4-a92e-4db7-b132-8166c46cc34a)

So once you click this, you'll see:
![image](https://github.com/user-attachments/assets/0356fa6c-2f4a-4ff1-949b-eb7a4a09ee31)

Here's the md of this anyways:

```md
| Software   | Description | Feature 1   | Feature 2   | ...   | Feature n   | Price   |
| :---        |    :----:   |    :----:   |    :----:   |    :----:   |    :----:   |          ---: |
| Anaplan   | Short desc | x |  | ... | x | $$$ |
| ...   | ... | ... | ... | ... | ... | ... ||
| Software n   | Short desc |  |  | ... | x | $$$ |
```